scripts/test/hive.sh

```diff
@@ -15,6 +15,13 @@
 # Wait until hive test server is ready, port 8899
 # is a status port indicates the hive server container
 # is ready, see .travis.yml for the details
+
+set -e
```

Comment:
Maybe we can add these in `travis.yaml` so that we can avoid `docker build`

https://github.com/sql-machine-learning/sqlflow/blob/7170c6f2702085961311ce1fb1fcf66bc0363a3a/.travis.yml#L36-L40

Reply:
> Maybe we can add these in `travis.yaml` so that we can avoid `docker build`
> 
> https://github.com/sql-machine-learning/sqlflow/blob/7170c6f2702085961311ce1fb1fcf66bc0363a3a/.travis.yml#L36-L40

I've tried a bit and it seems that travis lacks some `skip commits` mechanism. Any suggestion for how to do this?